about.css

```diff
@@ -0,0 +1,602 @@
+  margin-left: 2.5%;
+}
+
+.cont-contactinfo {
+  padding-top: 4vh;
+  padding-bottom: 4vh;
+  background-color: #e6e6ea;
+}
+
+.cont-contactinfo p {
+  font-size: 12px;
+  line-height: 24px;
+  text-align: center;
+}
+
+.cont-previousSummit p {
+  color: #969494;
+  font-size: 12px;
+  line-height: 20px;
+  width: 90%;
+  text-align: center;
+  margin-left: 5%;
+  margin-bottom: 7vh;
+}
+
+.cont-contactinfo p:nth-child(2) {
+  font-weight: 700;
+}
+
+.cont-logo {
+  background-color: #fff;
+}
+
+.cont-partners div {
+  display: flex;
+  justify-content: center;
+}
+
+.cont-logo div:first-child {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: center;
+  height: auto;
+}
+
+.cont-logo div:nth-child(2) h4:first-child,
+.cont-logo div:nth-child(2) h4:nth-child(2) {
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 30px;
+  text-align: center;
+}
+
+.cont-logo div:nth-child(2) h4:first-child {
+  padding-top: 3vh;
+  width: 5vw;
+}
+
+.cont-logoinfo {
+  padding-bottom: 4vh;
+}
+
+.cont-logo div:nth-child(2) {
+  display: flex;
+  justify-content: center;
+  background-color: white;
+  padding-bottom: 2vh;
+}
+
+.cont-previousSummit div hr {
+  width: 5vw;
+  height: 3px;
+  background-color: rgb(153, 88, 88);
+  border-color: rgb(153, 88, 88);
+  margin-top: 2vh;
+  margin-bottom: 2vh;
+}
+
+.cont-logo div:nth-child(2) hr {
+  width: 5vw;
+  height: 3px;
+  background-color: rgb(153, 88, 88);
+  border-color: rgb(153, 88, 88);
+  margin-top: 3vh;
+  margin-bottom: 2vh;
+}
+
+.cont-previousSummit h5 {
+  color: black;
+  font-size: 20px;
+  text-align: center;
+}
+
+.cont-partners div h5 {
+  font-size: 20px;
+  font-weight: 400;
+  color: #fff;
+  padding-top: 3vh;
+  padding-bottom: 1vh;
+}
+
+.cont-logoinfo h5:first-child {
+  padding-top: 2vh;
+  color: #969494;
+  text-align: center;
+}
+
+.cont-conferenceLogo img {
+  width: 100%;
+}
+
+.past-conference {
+  background-color: #fff;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.cont-previous div:nth-child(2) {
+  display: flex;
+  flex-direction: column;
+}
+
+.cont-partners div:nth-child(2) {
+  padding-bottom: 4vh;
+}
+
+.cont-partners div:nth-child(3),
+.cont-partners div:last-child {
+  width: 100vw;
+  display: flex;
+  justify-content: space-around;
+  padding-bottom: 3vh;
+}
+
+.cont-rights div:first-child {
+  grid-area: 1/1/2/2;
+  width: 30vw;
+  margin-left: 12vw;
+}
+
+.cont-rights div:last-child {
+  grid-area: 1/2/2/3;
+  width: 90%;
+  margin-left: 3vw;
+}
+
+.past-conference .cont-previous .cont-previousSummit div {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.cont-conferenceImages {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 3vh;
+}
+
+.cont-conferenceImages img {
+  width: 90%;
+}
+
+.cont-conferenceImages span {
+  position: absolute;
+  color: #fff;
+  width: 80%;
+  font-size: 20px;
+  font-weight: 700;
+  margin-top: 15vh;
+}
```

Comment:
- [ ] Kindly consider making sure that your content aligned is aligned with the center of the card as shown in the original design. This is because you have a bit of margin on top of your card content. See below for more details:
<table>
<tr>
<th align=“middle”>Behance design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="848" alt="Screenshot 2023-08-03 at 08 56 37" src="https://github.com/eamador1/summit/assets/107192076/f2f2ff32-6ef8-4a2c-a381-6d2dd4d50532">
</td>
<td>
<img width="1439" alt="Screenshot 2023-08-03 at 08 57 07" src="https://github.com/eamador1/summit/assets/107192076/799fd788-a14d-41b3-a541-165747c16c08">
</td>
</tr>
</table>